### ISSUE TITLE
Oxidization of performance sensitive tools

### ISSUE THREAD

**Issue by Selyss** [fosslife/devtools-x#54]
Hello, I would like to help with converting some performance sensitive features to Rust as mentioned in the README. Is there a To-Do list of these modules that would benefit from being converted?

**Comment by Sparkenstein** [comment]
I did some analysis, currently most of the modules that needed oxidization are already in rust, ex: image compression with SIMD etc. rest there are some modules which use heavy processing like random text generator, but they are still in JS land anyway, mainly because I did stress testing with them, most of them work just fine for huge workload.

rewriting these modules might save us time in 'calculation', but it will add extra latency in IPC (from rust to js) which is not worth the cost. tauri IPC is costly, 

Feel free to benchmark any JS modules you want that are not in rust, we do not want to target extreme cases like generating 1Million random strings, no user will be doing that in day-to-day life.  If you find something is super slow with JS we can definitely try rust
